src/parser/priest/holy/CombatLogParser.tsx

```diff
@@ -139,6 +142,9 @@ class CombatLogParser extends CoreCombatLogParser {
 
     // Conduits
     resonantWords: ResonantWords,
+
+    // Conduits
```

Comment:
Comment should be Covenants here for consistency

Reply:
Fixed